objectRecognitionSource/src/test/java/io/vantiq/extsrc/objectRecognition/neuralNet/TestYoloProcessor.java

```diff
@@ -36,6 +36,8 @@
     static final String PB_FILE            = "yolo.pb";
     static final String OUTPUT_DIR         = "src/test/resources/out";
     static final int    SAVE_RATE          = 2; // Saves every other so that we can know it counts correctly
+    static final String AUTH_TOKEN         = "Ax6jSjrrwqnaHmZtWhXvVV1ym8ARfpta6wwmuPhy928=";
```

Comment:
I'm assuming that this means we are dependent upon dev (or test or something) being up for testing.  We'll need (not required now) to improve this so that it sets up the test on its own.  Otherwise, there's a _hidden dependency_ on this one account with some set of stuff set up.